pyproject.toml

```diff
@@ -3,3 +3,49 @@ target-version = ['py39']
 extend-exclude = '''
     migrations
 '''
+
+[tool.poetry]
+name = "guide"
+version = "0.1.0"
+description = "Wagtail User Guide"
+authors = [
+    "Hitansh Shah <shah.hitanshsanjay.mat20@itbhu.ac.in>",
+    "Thibaud Colas <thibaudcolas@gmail.com>",
+    "Coen van der Kamp <coen@fourdigits.nl>",
```

Comment:
I couldn't find Meagen's email, please let me know so I can add her.

Reply:
ping @vossisboss

Reply:
We have plans to list contributors on the actual site so I’ll remove authors from here anyway.